docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -30,12 +30,12 @@ node:
 
 When you start a master-eligible node, you can provide this setting on the
 command line or in the `elasticsearch.yml` file. After the cluster has formed,
-this setting is no longer required and is ignored. It need not be set on
-master-ineligible nodes, nor on master-eligible nodes that are started to join
-an existing cluster. Note that master-eligible nodes should use storage that
-persists across restarts. If they do not, and `cluster.initial_master_nodes` is
-set, and a full cluster restart occurs, then another brand-new cluster will
-form and this may result in data loss.
+this setting is no longer required and is ignored. It should not be set on
```

Comment:
Avoid using `nor` or future tense when possible.

Consider making this a list.

```
It should not be set for:

* Master-ineligible nodes
* Master-eligible nodes joining an existing cluster
* Cluster restarts
```

If you prefer a sentence with a serial, I recommend this:

```
It should not be set for master-ineligible nodes,
master-eligible nodes joining an existing cluster,
or cluster restarts.
```

Reply:
I like `nor` :) I have implemented the one-sentence version, because this doesn't seem important enough to warrant a list.